### PR TITLE
Python job

### PR DESCRIPTION
**Pull request by mebster**

### CODE REVIEW

File: _jobs/python-software-developer.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+- Experience in Docker and/or Kubernetes
+- Experience of a shell scripting language (e.g. bash)
+- Good knowledge of Linux (NuvlaEdge micro-services are often based on Ubuntu)
+
+
+**Nice to have**
+
+- Experience with micro-service architecture (NuvlaEdge is a set of Docker images orchestrated using DockerSwarm or Kubernetes)
+- Experience with embedded system development
+- Knowledge of service operations in cloud environments - e.g. Exoscale, AWS, Azure, GCE
```
Comment (konstan) [inline _jobs/python-software-developer.md:54]: Suggestion to change: "e.g. Exoscale, AWS, Azure, GCE" -> "e.g. AWS, Azure, GCE, CloudStack, OpenStack"

File: _jobs/python-software-developer.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+- Experience in working in an open source environment
+- Experience in contributing to open source projects
+- Experience with GitHub and its eco-system
+- Functional language programming experience (e.g. Clojure/Clojurescript, Scala, Haskell)
```
Comment (konstan) [inline _jobs/python-software-developer.md:58]: Clojurescript -> ClojureScript

File: _jobs/python-software-developer.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+### Description
+
+SixSq is building the most advanced edge-to-cloud B2B SaaS platform: Nuvla.io.  With the recent acquisition of SixSq by Ekinops, we are growing. We are looking for software developers, wanting to join a multi-cultural and international team.  As a software developer you will contribute to the overall objective of the company, which is to disrupt the IT industry with an open, agile, simple and efficient platform. The platform also includes a marketplace of business applications targeting the far edge in a range of sectors such as retail, manufacturing, transport, telecoms and energy.  In time, since we sell as a team, you will also be asked to support sales efforts via creative demonstrations and technical discussions with customers and prospects.
+
+To communicate with Nuvla.io, edge devices run the NuvlaEdge software.  Packaged as micro-services (e.g. Docker and Kubernetes), NuvlaEdge provides the local environment that allows Nuvla.io to manage fleets of edge devices, as well as the life-cycle of business apps running at the edge.
```
Comment (konstan) [inline _jobs/python-software-developer.md:13]: "Packaged as micro-services (e.g. Docker and Kubernetes)," -> "Designed and implemented as microservices and running on Docker and Kubernetes,"

File: _jobs/python-software-developer.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+**Nice to have**
+
+- Experience with micro-service architecture (NuvlaEdge is a set of Docker images orchestrated using DockerSwarm or Kubernetes)
+- Experience with embedded system development
+- Knowledge of service operations in cloud environments - e.g. Exoscale, AWS, Azure, GCE
+- Experience in working in an open source environment
+- Experience in contributing to open source projects
+- Experience with GitHub and its eco-system
+- Functional language programming experience (e.g. Clojure/Clojurescript, Scala, Haskell)
+- Understanding and experience of OpenVPN
```
Comment (konstan) [inline _jobs/python-software-developer.md:59]: Understanding of and experience with OpenVPN.

File: _jobs/python-software-developer.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+- Ability to communicate effectively with internal teams, partners and customers
+- Experience or interest in agile methodologies (e.g. Scrum, Kanban, DevOps)
+- Experience in Docker and/or Kubernetes
+- Experience of a shell scripting language (e.g. bash)
+- Good knowledge of Linux (NuvlaEdge micro-services are often based on Ubuntu)
+
+
+**Nice to have**
+
+- Experience with micro-service architecture (NuvlaEdge is a set of Docker images orchestrated using DockerSwarm or Kubernetes)
```
Comment (konstan) [inline _jobs/python-software-developer.md:52]: micro-service -> microservice

**Review by konstan — APPROVED** [review]
I provided some minor comments. But in overall, all looks good to me.

**Review by schaubl — APPROVED** [review]

**Pull request merged**